doc/rst/source/tutorial/julia/index.rst

```diff
@@ -25,10 +25,18 @@ still redirect to the hard-core GMT man pages, whilst others direct users to Jul
 the translated GMT manuals. A *lost case* is the GMT Technical Reference that is so big/complete that it
 will take long time to see a Julia version of it.
 
+See the :doc:`install guide </install>`
+for instructions and to make sure you have all required dependencies installed.
+Alternatively, you can build GMT from source by following the
+`building guide <https://github.com/GenericMappingTools/gmt/blob/master/BUILDING.md>`_.
+
+In addition, we recommend access to any flavor of the UNIX operating system
```

Comment:
I think this paragraph does not belong here. The Julia command line provides all that is needed.

Reply:
Yes. Now fixed.